### PR TITLE
Update file-explorer.md

### PR DESCRIPTION
**Pull request by crutkas**
https://github.com/microsoft/PowerToys/issues/1796

### CODE REVIEW

**Comment by PRMerger19** [comment]
@crutkas : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: hub/powertoys/file-explorer.md [hunk 1]
```diff
@@ -23,27 +23,38 @@ File Explorer add-ons currently include:
...
+
+To enable, set which you would like enabled to **On**.
 
 ![PowerToys Settings Enable File Explorer screenshot.](../images/powertoys-settings-fileexplorer.png)
 
+### Enabling Preview Pane in File Explorer on Windows
+
 > [!NOTE]
-> Windows Explorer has an additional setting that needs to be checked in order for preview handlers to work. Open Explorer's Folder options, go to tab "View", under "Advanced settings" check **Show preview handlers in preview pane**.
+> There is an advanced setting in Windows that may be unchecked. If previewers don't seem to work, including ones like jpeg images, please verify this setting is checked. Open Explorer's Folder options, go to tab "View", then under "Advanced settings" check **Show preview handlers in preview pane**.
```
Comment (Jay-o-Way) [inline hub/powertoys/file-explorer.md]:
```suggestion
> There is an advanced setting in Windows that might be turned off. If preview handlers (including ones like jpeg images) don't seem to work, please verify this setting is turned on. Open Explorer's Folder options, go to tab "View", then under "Advanced settings" check **Show preview handlers in preview pane**.
```

I've seen this "may versus might" in other docs actively being changed. ("may" implies permission, "might" implies chance)
A _setting_ is not really _unchecked_ - at least not in the way you mean it (a UI control would be)
Also: "previewers" sounds like you created a new word.

**Comment by Court72** [comment]
@mattwojo 

Can you review the proposed changes? *IMPORTANT*: When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

File: hub/powertoys/file-explorer.md [hunk 2]
```diff
@@ -23,27 +23,38 @@ File Explorer add-ons currently include:
...
-## Preview Pane
 
-Preview Pane is an existing feature in the Windows File Explorer which shows a lightweight, rich, read-only preview of the file's contents in the view's reading pane. PowerToys adds four extensions: Markdown, SVG, PDF, and G-code. PowerToys also adds support for source code files (for more than 150 file extensions).
+## Thumbnail preview
+
+Showing thumbnails is a built-in Windows feature. For thumbnail preview, PowerToys adds multiple extensions: SVG, PDF, G-code and STL.
+
+### Enabling PowerToys Thumbnail previews
+
+To enable, set which you would like enabled to **On**.
```
Comment (Jay-o-Way) [inline hub/powertoys/file-explorer.md]:
```suggestion
To enable, set those which you would like enabled to **On**.
```

File: hub/powertoys/file-explorer.md [hunk 3]
```diff
@@ -23,27 +23,38 @@ File Explorer add-ons currently include:
...
+
+## Preview Pane previewers in PowerToys
 
-### Enabling Preview Pane
+Preview Pane is an existing feature in the Windows File Explorer which allows you to see a preview of the file's contents in the view's reading pane. PowerToys adds multiple extensions: Markdown, SVG, PDF, and G-code. In addition to those, PowerToys also adds support for source code files (for more than 150 file extensions).
 
-To enable, first set all to **On**.
+### Enabling PowerToys Preview Pane previewers
+
+To enable, set which you would like enabled to **On**.
```
Comment (Jay-o-Way) [inline hub/powertoys/file-explorer.md]:
```suggestion
To enable, set those which you would like enabled to **On**.
```

**Review by Jay-o-Way — CHANGES_REQUESTED** [review]
Couple of text suggestions

File: hub/powertoys/file-explorer.md (hunk 2 above)
Comment (crutkas) [reply hub/powertoys/file-explorer.md]: @mattwojo 'set those which' sounds awkward to me, thoughts?

File: hub/powertoys/file-explorer.md (hunk 3 above)
Comment (crutkas) [reply hub/powertoys/file-explorer.md]: @mattwojo 'set those which' sounds awkward to me, thoughts?

File: hub/powertoys/file-explorer.md (hunk 2 above)
Comment (crutkas) [reply hub/powertoys/file-explorer.md]: set which items you ....?

File: hub/powertoys/file-explorer.md (hunk 3 above)
Comment (crutkas) [reply hub/powertoys/file-explorer.md]: set which items you ....?

**Comment by PRMerger6** [comment]
@crutkas : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by crutkas** [comment]
@mattwojo can this get approved and merged

File: hub/powertoys/file-explorer.md (hunk 2 above)
Comment (mattwojo) [reply hub/powertoys/file-explorer.md]: Updating to: To enable preview support, set the extension to **On**.

File: hub/powertoys/file-explorer.md (hunk 3 above)
Comment (mattwojo) [reply hub/powertoys/file-explorer.md]: Updating to: "To enable preview support, set the extension to **On**."

**Comment by PRMerger13** [comment]
@mattwojo : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Review by mattwojo — APPROVED** [review]
Looks good.

**Pull request merged**